homepage/homepage/app/(docs)/docs/[framework]/[...slug]/groups/inheritance.mdx

```diff
@@ -0,0 +1,161 @@
+import { ContentByFramework, CodeGroup } from '@/components/forMdx'
+
+export const metadata = { title: "Group inheritance" };
+
+# Group Inheritance
+
+Groups can inherit members from other groups using the `extend` method. 
+
+When a group extends another group, members of the parent group will become automatically part of the child group.
+
+## Basic Usage
+
+Here's how to extend a group:
+
+<CodeGroup>
+```typescript
+const playlist = Group.create();
+const track = Group.create();
+
+// This way track becomes visible to the members of playlist
+track.extend(playlist);
+```
+</CodeGroup>
+
+When you extend a group:
+- Members of the parent group get access to the child group
+- Their roles are inherited (with some exceptions, see [below](#role-inheritance-rules))
+- Removing a member from the parent group also removes their access to child groups
+
+## Setting the extension role
+
+You can set the extension role by passing a second argument to `extend`:
+
+<CodeGroup>
+```typescript
+const organization = Group.create();
+
+const billing = Group.create();
+
+// This way the members of the organization can only read the billing data
+billing.extend(organization, "reader"); 
+```
+</CodeGroup>
+
+The extension role always overrides the role of the accounts in the parent group:
```

Comment:
Now that we explain this in the intro we can combine the preceding and the following listing and delete this sentence

Reply:
Changes this sentence and the example into:

```
The "override role" works in both directions:

<CodeGroup>
  ```typescript
  const parentGroup = Group.create();
  parentGroup.addMember(bob, "reader");
  parentGroup.addMember(alice, "admin");
  
  const childGroup = Group.create();
  childGroup.extend(parentGroup, "writer");
  
  // Bob and Alice are now writers in the child group
 /```
</CodeGroup>
```

Reply:
I might look redundant, but I want to make this more clear and preserve the billing example which is the main driver of the whole feature.